### PR TITLE
TILA-426 | ApplicationEvent api data enrichment

### PR DESCRIPTION
**Pull request by nicobav**
Adds purpose name, ability group name and reservation unit details into application event data returned by api

### CODE REVIEW

**Comment by nicobav** [comment]
Depends on changes in https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/96

File: api/applications_api/serializers.py [hunk 1]
```diff
@@ -298,6 +315,14 @@ class Meta:
             },
         }
 
+    def get_purpose_name(self, obj):
```
Comment (aprenfors) [inline api/applications_api/serializers.py:318]: Not 100% related to this PR, but purpose name doesn't seem to be unique, which it most likely should be. While this is read only thingy, i think when used like this should be maybe used only on unique values to be super clear. 

But no need to do anything in this pr, more or less for discussion.

**Review by aprenfors — APPROVED** [review]

File: api/applications_api/serializers.py (hunk 1 above)
Comment (nicobav) [reply api/applications_api/serializers.py:318]: Yes, I think the purposes should be perhaps be pre defined => it could be just enum also

Comment (aprenfors) [reply api/applications_api/serializers.py:318]: Not sure if it can be an enum, might need some flexibility. Anyway this looks fine to me.

**Pull request merged**